src/autocomplete.js

```diff
@@ -179,7 +179,8 @@ export default class Autocomplete {
 
   fetchResults() {
     const query = this.input.value.trim()
-    if (!query) {
+
+    if (!query && !this.container.fetchWhenBlank) {
```

Comment:
Do we really need `fetchWhenBlank` getter/setter? I don't see this having programmatic usage. We could simply do `container.hasAttribute('...')` here.

Reply:
No I think you're right, removed.